gosec/gosec.js

```diff
@@ -19,14 +19,16 @@ module.exports = (directory, inputFiles, reportFile) => {
   }
   reportFile = reportFile || 'gosec.json'
   const currentDirectory = path.join(__dirname, '../', directory)
+
   /**
   * @argument gosec command which the child process will execute
   * @argument -fmt output format of the command
   * @argument out flag which redirects the gosec output to a file
   * @argument reportFile the file where the output of gosec will be stored
   * @argument goFiles files which will be analyzed by gosec
   */
-  let gosecArgs = ['-fmt=json', '-out', reportFile, goFiles]
```

Comment:
The javascript spread operator `...` does exactly this (see the bandit runner).

Reply:
Yep, you were right. I change my code to use this operator.